boa3_test/tests/compiler_tests/test_file_generation.py

```diff
@@ -1164,3 +1163,13 @@ def verify_parameters_and_return_manifest(self, path: str) -> Tuple[dict, list]:
             self.assertIn('returntype', abi_method)
 
         return methods, abi['methods']
+
+    def test_event_runtime_notify_manifest(self):
+        path = self.get_contract_path('ManifestEventRuntimeNotify.py')
+        nef_output, expected_manifest_output = self.get_deploy_file_paths_without_compiling(path)
+        compiler = Compiler()
+        with LOCK:
+            compiler.compile_and_save(path, nef_output)
+            _, manifest = self.get_output(nef_output)
+
+        self.assertTrue(len(manifest['abi']['events']) > 0)
```

Comment:
Since this seems to be a "notify" specific problem, it would be better to also check if the event on the manifest is named "notify"